test/python/test_variableops_static_input.py

```diff
@@ -159,11 +159,6 @@ def test_variable_static_input_variables_share_buffer(self):
         env_var_map = self.store_env_variables([buffer_sharing_env])
         self.set_env_variable(buffer_sharing_env, "1")
 
-        # disable NoOp it forms a single cluster with no inputs or outputs
-        # Pipelined Tensor Store thinks it is depth 0 and errors out
```

Comment:
there are two tests in this file. please do it for both and in clean up we are also setting ngraph_bridge.set_disabled_ops('')

I think there will be 4 lines change in this file

Reply:
ya, you need to remove here as well - https://github.com/tensorflow/ngraph-bridge/blob/master/test/python/test_variableops_static_input.py#L118